coderkevin/fresh-data
# Update config for npm

This prepares the build to be distributed via npm.

To Test:
1. `npm install`
2. `npm pack`

To test hello-world example:
1. `npm link`
2. `cd examples/hello-world`
3. `npm install`
4. `npm link fresh-data`
5. `npm start`

You should have a resulting `.tgz` file which contains `es` and `lib`, each with trees of transpiled files, and a `dist` folder with `fresh-data.js` and `fresh-data.min.js` bundles.

P.S. Don't be scared by the high line count, there are a lot of package-lock.js lines in there. There are only 223 actual lines added.
